metricflow/model/semantics/semantic_model_lookup.py

```diff
@@ -202,7 +205,11 @@ def _add_semantic_model(self, semantic_model: SemanticModel) -> None:
 
         for measure in semantic_model.measures:
             self._measure_aggs[measure.reference] = measure.agg
-            self._measure_index[measure.reference].append(semantic_model)
+
+            semantic_models_for_measure = self._measure_index.get(measure.reference, [])
+            semantic_models_for_measure.append(semantic_model)
+            self._measure_index[measure.reference] = semantic_models_for_measure
```

Comment:
```python
self._measure_index[measure.reference] = self._measure_index.get(measure.reference, []) + [semantic_model]
```
Just throwing this one out there as a possibility, pick whichever you'd prefer tho!

Reply:
good idea!